usr/src/uts/common/inet/ip/ip_if.c

```diff
@@ -1394,10 +1394,11 @@ ill_capability_wait(ill_t *ill)
 
 	while (ill->ill_capab_pending_cnt != 0 &&
 	    (ill->ill_state_flags & ILL_CONDEMNED) == 0) {
-		mutex_enter(&ill->ill_dlpi_capab_lock);
+		/* This may enable blocked callers of ill_capability_done(). */
 		ipsq_exit(ill->ill_phyint->phyint_ipsq);
-		cv_wait(&ill->ill_dlpi_capab_cv, &ill->ill_dlpi_capab_lock);
-		mutex_exit(&ill->ill_dlpi_capab_lock);
+		/* Pause a bit (1msec) before we re-enter the squeue. */
+		delay(drv_usectohz(1000000));
```

Comment:
How do we know pausing will solve the original issue and not just make it less likely to happen?

Reply:
If "original issue" == kernel panic, it's because we're not holding a mutex anymore prior to calling ipsq_exit().

If "original issue" == OS-4683, we'll need to re-run the OS-4683 tests to be sure.  I *did* receive a "seems okay" from original OS-4683 author Cody Mello today, but running those tests will be further proof.

Reply:
OS-4683 test I ran completed successfully (and showed no difference vs. a stock PI).